### ISSUE TITLE
Feature Request: Heading control in precision landing

### ISSUE THREAD

**Issue by derekxm** [diydrones/ardupilot#3328]
Precision landing should be able to control the heading as well as the position of the copter. For example, if the copter is trying to landing on a battery swapping station, the heading of the copter is very important for battery swapping mechanism to engage correctly.

**Comment by magicrub** [comment]
This is being addressed here: https://github.com/mavlink/mavlink/pull/432

**Comment by rmackay9** [comment]
I think it'll take some changes on the copter side as well though so let's leave this one open I think.

**Comment by Naterater** [comment]
Does precision landing control heading yet?

**Comment by rmackay9** [comment]
No it doesn't I'm afraid

**Comment by IamPete1** [comment]
@rishabsingh3003 is this issue still valid?

**Comment by rishabsingh3003** [comment]
@IamPete1 yes this issue is still valid. At the moment we expect users to control yaw manually via RC. What does the user want? The heading should point towards the direction of where the copter is going (i.e point dront towards landing target)? I think this is a one-line trivial change and should be easy to do... otherwise, we can always control yaw via Lua/Mavlink scripts..
